### PR TITLE
Refactor of validation module

### PR DESCRIPTION
**Pull request by rhohndorf**
* Removed unsed imports
* Changed imports from relative to module style
* Removed unnecessary files
* Renamed modules, functions and instances following PEP8
* Changed validate function to expect RequestData and return ResponseData object

### CODE REVIEW

File: resources/tiger_pcl.h5
Comment (the-mx) [inline resources/tiger_pcl.h5:1]: Mock file for tests? Should we move it somewhere inside the validation folder?

File: validation/serve.py [hunk 1]
```diff
@@ -18,36 +20,43 @@ def get_args():
...
-    print("[INFO] Start validating the input 3D data.")
-    print("[INFO] Input prompt: ", message.prompt_in)
 
+class RequestData(BaseModel):
+    prompt: str
+    data: str
+
+
+class ResponseData(BaseModel):
+    scores: List[float]
```
Comment (the-mx) [inline validation/serve.py:30]: If we give only one 3D object as an input, we should get only score as the output

File: validation/serve.py [hunk 2]
```diff
@@ -18,36 +20,43 @@ def get_args():
...
-    Validator.init_gaussian_splatting_renderer()
-    scores = Validator.score_response_gs_input([message], save_images=save_images, cam_rad=4)
+    t23D = protocol.TextTo3D(prompt_in=request.prompt, mesh_out=request.data)
+    validator = TextTo3DModelValidator(512, 512, 10)
+    validator.init_gaussian_splatting_renderer()
+    scores = validator.score_response_gs_input([t23D], save_images=False, cam_rad=4)
 
     t2 = time()
     print("[INFO] Score: ", scores)
     print("[INFO] Validation took: ", t2 - t1, " sec")
```
Comment (the-mx) [inline validation/serve.py:56]: I'd prefer:
f"[INFO] Validation took: {t2 - t1} sec"

File: README.md [hunk 3]
```diff
@@ -102,7 +102,29 @@ Validators update the miner list hourly. It's normal not to observe requests in
...
+### Setup
+To install all dependencies and generate the run configuration execute the following command in the project base directory:
+```
+make validator
+```
+
+### Startup
+To start the validator service execute the following command in the project base directory:
+```
+make start_validator
```
Comment (the-mx) [inline README.md:116]: Running this leads to `ModuleNotFoundError: No module named 'neurons'`

File: validation/serve.py (hunk 1 above)
Comment (rhohndorf) [reply validation/serve.py]: Done

File: tests/test_validation.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import base64
+
+import h5py
+import pytest
+
+from neurons.old.protocol import TextTo3D
```
Comment (the-mx) [inline tests/test_validation.py:6]: This process should be protocol agnostic. Just use binary data

File: validation/serve.py (hunk 2 above)
Comment (rhohndorf) [reply validation/serve.py]: Done. Changed it for all three print statements that took multiple arguments to keep things uniform.

Comment (rhohndorf) [reply validation/serve.py]: Also I would use a logger instead of prints for this.

File: resources/tiger_pcl.h5
Comment (rhohndorf) [reply resources/tiger_pcl.h5:1]: I think we should separate code and data. I could put it in resources/validation/
what do you think about that ?

Comment (the-mx) [reply resources/tiger_pcl.h5:1]: ok

File: tests/test_validation.py (hunk 4 above)
Comment (rhohndorf) [reply tests/test_validation.py:6]: Removed usage of TextTo3D from code in validation

File: README.md (hunk 3 above)
Comment (rhohndorf) [reply README.md:116]: Should also solved

**Review by the-mx — APPROVED** [review]

**Pull request merged**